bind/bind.go

```diff
@@ -53,6 +53,44 @@ func GenCPython(w io.Writer, fset *token.FileSet, pkg *Package, lang int) error
 	return err
 }
 
+/**
```

Comment:
the canonical way to add documentation for a Go type, func, var, const, ... is to use `// `.
yeah, even for multi-lines comments.